### ISSUE TITLE
organism detection datum hierarchy

### ISSUE THREAD

**Issue by zhengj2007** [VEuPathDB-ontology/VEuPathDB-ontology#480]
We moved presence data items under the corresponding organism detection data item. 
1. Do we need to deprecate the terms, such as 'presence of organism in blood' and 'presence of organism in cerebrospinal fluid'?
2. Shall we organize the terms based on the NCBITaxon hierarchy? such as, specify that 'presence of Cryptococcus in cerebrospinal fluid' is a subClassOf 'presence of eukaryote in cerebrospinal fluid', 'presence of Plasmodium in blood' is a subClassOf 'presence of eukaryote in blood', etc.?
Complicated case is 'presence of Plasmodium in blood by thick smear microscopy assay', I think it is the subClassOf both 'presence of Plasmodium in blood' and 'presence of Plasmodium in blood by microscopy assay'. Need to check when OBI update the logic axioms of 'organism identification datum', what VEuPathDB terms should change. May need to define some classes using equivalent axioms.
3. The terms are classified by:
- organism (bacteria, virus, etc.) in specimen for detection: is_about some NCBITaxon
- specimen type: is_specificed_output_of some 'organism detection assay' that indicate the specimen type as input
- detection method: is_specificed_output_of some 'detection assay' (PCR, ELISA, MICROSCOPY)
- organism role (mother, child) that specimen taken from: something like is_specificed_output_of some ' 'mother/child organism detection assay', for this purpose, we need to add some specimen terms as: maternal blood specimen, and person terms like 'biological mother of participant under investigation'
- raw result or boolean result: use boolean value to distinguish the terms, have few detection data in boolean value but not mean present or absent. We may implement distinction using other axioms.

Need to create a design pattern to be able to classify the data terms based on all above criteria.

**Comment by johnwjudkins** [comment]
1. Yes.
2. Maybe. After some thought, there may be a difference between the leaf presence terms in ClinEpi and the category terms. For example:

EUPATH:0032207 (leaf) is the class of any Boolean data on E. coli (no more or less specific than the species) detection in a blood specimen.
Its ClinEpi parent term EUPATH:0033298 (category) is the class of any Boolean data on Escherichia (mentioning the genus only or the species, strain, etc.)

So, I think the leaf classes could be subclasses of the category classes, just as they are in the project .owl files, if they are all defined according to the above.

**Comment by johnwjudkins** [comment]
I agree to deprecate [presence of organism in specimen](http://purl.obolibrary.org/obo/EUPATH_0000762), but it has children, described here: https://github.com/VEuPathDB-ontology/VEuPathDB-ontology/issues/414

**Comment by zhengj2007** [comment]
If we keep the term, we'd think update design patterns as I mentioned in the tracker to make the terms have correct parents after reasoning. For example, define [presence of organism in specimen](http://purl.obolibrary.org/obo/EUPATH_0000762) = 'organism detection data' and 'has value specification' some ('has specified numeric value' only xsd:boolean)

**Comment by johnwjudkins** [comment]
The relation should still be 'has specified value' (not numeric).

**Comment by johnwjudkins** [comment]
> If we keep the term, we'd think update design patterns as I mentioned in the tracker to make the terms have correct parents after reasoning. For example, define [presence of organism in specimen](http://purl.obolibrary.org/obo/EUPATH_0000762) = 'organism detection data' and 'has value specification' some ('has specified numeric value' only xsd:boolean)

'presence of organism in specimen' is _not_ equivalent to ('organism detection datum' and 'has value specification' some ('has specified value' only xsd:boolean). This is because there can exist other types of Boolean data under 'organism detection datum'. An example is whether the number of parasites in a specimen is greater or less than some threshold value.

**Comment by johnwjudkins** [comment]
We'd need something like this (may need help from PATO):

'presence of organism in specimen' equivalentTo
'organism detection datum' and 'is about' only (('absence of physical object' or present) and ('quality of' some specimen) and (toward some organism))

**Comment by johnwjudkins** [comment]
I submitted a pull request that changes the raw data axioms to one equivalent axiom
Example: EUPATH:0000537 is logically defined as
'eukaryota in blood datum' and ('is about' some 'Plasmodium') and (is_specified_output_of some 'microscopy assay').

Reasoning is fast and produces examples like the following:

'Plasmodium in umbilical cord blood by microscopy assay datum' is inferred as subclass of 'Plasmodium in blood by microscopy assay datum'.
'qPCR threshold cycle for Plasmodium falciparum in blood' is inferred as subclass of 'Plasmodium in blood by real time polymerase chain reaction assay datum'.

**Comment by johnwjudkins** [comment]
> We'd need something like this (may need help from PATO):
> 
> 'presence of organism in specimen' equivalentTo 'organism detection datum' and 'is about' only (('absence of physical object' or present) and ('quality of' some specimen) and (toward some organism))

Thought about it more. We don't need the PATO terms. We can logically distinguish a presence of Plasmodium term from its analogous raw data term by adding this axiom, along with the Boolean axiom:

'is about' only (('has part' some Plasmodium) or not ('has part' some Plasmodium))

to indicate whether the specimen contains Plasmodium or not.
